docs/locators.md

```diff
@@ -0,0 +1,289 @@
+# Locators
+
+"Locators" in `wdi5` are referred to [as "Matchers" in OPA5](https://ui5.sap.com/#/api/sap.ui.test.matchers): they identify a UI5 control to retrieve and use for further (test-)work.  
+At the same time, locators are a subset [of a "Selector" in `wdi5`](usage#control-selectors).
+
+```javascript
+// this is the overall selector
+const identifier = {
+  wdio_ui5_key: "someCustomIdInWdi5Test",
+  forceSelect: true,
+  // this is the locator/matcher (sorry for not naming it as such 😬)
+  selector: {
+    i18NText: {
+      propertyName: "text",
+      key: "startPage.navButton.text"
+    },
+    controlType: "sap.m.Button",
+    viewName: "test.Sample.view.Main"
+  }
+}
+
+// get it
+const control = await browser.asControl(identifier)
+// use it
+await control.press()
+```
+
+Here are couple of samples to get you comfortable with `wdi5`'s/OPA5's locators/matchers.
+
+?> Note: all of them are essentially taken from the `/examples/ui5-js-app/webapp/test/e2e/**/*.test.js` files.
+
+## ancestor
+
+Get a "child" control from a "parent" (ancestor).
+
+```javascript
+it("get the child", async () => {
+  const ancestorSelector = {
+    selector: {
+      controlType: "sap.m.Title",
+      ancestor: {
+        controlType: "sap.m.Panel"
+      }
+    }
+  }
+
+  const text = await browser.asControl(ancestorSelector).getText()
+
+  expect(text).toEqual("Custom Toolbar with a header text")
+})
+```
+
+## bindingPath
+
+Retrieve a UI5 control that has a binding context with this exact binding path.  
+Typically not used standalone, but in combination with [the `controlType` locator](#controlType).
+
+```javascript
+it("select ui5 input control by binding and set new value (w/ custom wdio_ui5_key)", async () => {
+  const inputSelector = {
+    selector: {
+      interaction: "focus",
+      bindingPath: {
+        propertyPath: "/Customers('TRAIH')/ContactName"
+      },
+      viewName: "test.Sample.view.Main",
+      controlType: "sap.m.Input"
+    }
+  }
+
+  const value = "new value"
+  const input = await browser.asControl(inputSelector)
+  await input.enterText(value)
+
+  expect(await input.getValue()).toEqual(value)
+})
+```
+
+## controlType
+
+Specify a UI5 control by namespace for retrieval.  
```

Comment:
I think the description matches more the `viewName` locator and not the `controlType`^^

Reply:
re-worded this to make it more clear